CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.29.0](https://github.com/buildkite/agent/compare/v3.28.1...v3.29.0) (2021-04-21)
+
+### Changed
+* Support mips64le architecture target. [#1379](https://github.com/buildkite/agent/pull/1379) ([houfangdong](https://github.com/houfangdong))
+* Explicitly set a working directory when we start the bootstrap [#1402](https://github.com/buildkite/agent/pull/1402) ([yob](https://github.com/yob))
```

Comment:
maybe this could be under "fixed", as "avoid occasional failure to run jobs when working directory is missing"